File: 2st/Yeonwoo/배열 회전시키기.py
def solution(numbers, direction):
    answer = []
    if direction == "right":
        for i, num in enumerate(numbers):
            if i==len(numbers)-1:
                answer.insert(0,num)
            else:
                answer.insert(i+1, num)
    else:
        for i, num in enumerate(numbers):
            answer.insert(i-1,num)
    return answer

File: 4st/Jisoo/programmer/배열 회전시키기.java
class Solution {
    public int[] solution(int[] numbers, String direction) {
        int[] answer = new int[numbers.length];
        
        if(direction.equals("right")) {
            answer[0] = numbers[numbers.length - 1];
            
            for(int i = 1; i < numbers.length; i++) {
                answer[i] = numbers[i - 1];
            }
        } else {
            answer[numbers.length - 1] = numbers[0];
            
            for(int i = 0; i < numbers.length - 1; i++) {
                answer[i] = numbers[i + 1];
            }
        }
        
        return answer;
    }
}